components/financial_assistance/app/models/financial_assistance/application.rb

```diff
@@ -49,6 +50,35 @@ class Application # rubocop:disable Metrics/ClassLength TODO: Remove this
 
     UNEDITABLE_STATES = %w[cancelled renewal_draft].freeze
 
+    # @!attribute ORIGIN_SOURCE_KINDS
+    # @return [Array<Symbol>] Collection of all possible origin source kinds
+    ORIGIN_SOURCE_KINDS = %i[admin assister broker data_import migration system user].freeze
```

Comment:
- The term origin_source is redundant, as "origin" and "source" are similar in meaning.

@saikumar9 can this be renamed to ORIGIN_KINDS or SOURCE_KINDS?
```suggestion
    ORIGIN_KINDS = %i[admin assister broker data_import migration system user].freeze
```
or

```suggestion
    SOURCE_KINDS = %i[admin assister broker data_import migration system user].freeze
```